lib/newrelic.js

```diff
@@ -151,7 +151,9 @@ class NewRelic {
         };
 
         if (!this.canSendMetrics) {
-            console.error("Metrics not sent because disabled: ", metrics);
+            if(!process.env.OPENWHISK_NEWRELIC_DISABLE_METRICS) {
+                console.error("Metrics not sent because disabled: ", metrics);
+            }
```

Comment:
Might be clearer this way:

```
// at the start of the function, because building the `metrics` object also isn't needed
if (process.env.OPENWHISK_NEWRELIC_DISABLE_METRICS) {
    return;
}

if (!this.canSendMetrics) {
    console.error("Metrics not sent because disabled: ", metrics);
    return;
}
```

Also note we can just do `return` inside an `async` function, no need for `return Promise.resolve();`.

Reply:
well,
```js
if (!this.canSendMetrics) {
    console.error("Metrics not sent because disabled: ", metrics);
    return;
}
```
should be after building the metrics since we log the metrics